packages/react/src/components/Backdrop/Backdrop.stories.mdx

```diff
@@ -0,0 +1,65 @@
+import {ArgsTable, Source, Story, Canvas, Meta} from '@storybook/addon-docs';
+import {useArgs} from '@storybook/client-api';
+import dedent from 'ts-dedent';
+import StoryConfig from '../../../.storybook/story-config.ts';
+import Backdrop from './Backdrop.tsx';
+import Button from '../Button/Button.tsx';
+
+export const meta = {
+  component: Backdrop,
+  title: StoryConfig.Backdrop.hierarchy,
+};
+
+<Meta title={meta.title} component={meta.component} />
+
+export const Template = args => {
+    const [{open}, updateArgs] = useArgs();
+      return (
+        <>
+        <Button variant="outlined" onClick={() => updateArgs({open: !open})}>Show Backdrop</Button>
+            <Backdrop
+                sx={{ color: '#fff', zIndex: 9 }}
+                open={open}
+                onClick={() => updateArgs({open: !open})}
+                {...args}
+            >
+        </Backdrop>
+        </>
+    )
+};
+
+# Backdrop
+
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+
+## Overview
+
+The `Backdrop` component is used to provide emphasis on a particular element or parts of it.
+
+<Canvas>
+  <Story
+    name="Overview"
+    args={{
+      open: false,
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+## Props
+
+<ArgsTable story="Overview" />
+
+## Usage
+
+Import and use the `Backdrop` component in your components as follows.
+
+<Source
+  language="jsx"
+  dark
+  format
+  code={dedent`import Backdrop from '@oxygen-ui/react/Backdrop';\n`}
```

Comment:
Shouldn't we add example of using backdrop component here, just like https://github.com/wso2/oxygen-ui/blob/main/packages/react/src/components/Box/Box.stories.mdx#L60 ?